src/main/resources/application.yml

```diff
@@ -107,3 +107,13 @@ notify:
 self:
   link: "http://localhost:3000"
 
+hmpps:
+  prison:
+    api:
+      url: "https://api-dev.prison.service.justice.gov.uk/api"
+
+oauth:
+  client:
+    id: ${SYSTEM_CLIENT_ID}
+    secret: ${SYSTEM_CLIENT_SECRET}
+    token-url: ${HMPPS_AUTH_TOKEN_URL}
```

Comment:
think the token url is picked up automatically if put in a specific place? 
https://github.com/ministryofjustice/hmpps-welcome-people-into-prison-api/blob/main/src/main/resources/application.yml#L65